internal/console/interactive.go

```diff
@@ -21,12 +22,12 @@ var errExitTrap = errors.New("exit trap")
 
 type KongContextBinder func(ctx context.Context, kctx *kong.Context, projectConfig projectconfig.Config, app *kong.Kong, cancel context.CancelFunc) context.Context
 
-func RunInteractiveConsole(ctx context.Context, k *kong.Kong, projectConfig projectconfig.Config, binder KongContextBinder, refreshFunction func(func()), cancelContext context.CancelFunc) error {
+func RunInteractiveConsole(ctx context.Context, k *kong.Kong, projectConfig projectconfig.Config, binder KongContextBinder, refreshFunction func(func()), cancelContext context.CancelFunc, client ftlv1connect.ControllerServiceClient) error {
```

Comment:
Can you document what "refreshFunction" is? `func(func())` doesn't tell me much.

Reply:
Also it appears to be optional, so it should be behind an `optional.Option`.

Reply:
My latest PR deletes it.